CONTRIBUTING.md

```diff
@@ -66,8 +104,40 @@ Spaces are needed between function agruments (declaration and definition). Space
 ```
 ***
 
-Boolean variables and return values should use TRUE and FALSE constans instead of 1 and 0.
 
+Don't have unnecessary return calls or return meaningless data.
+If there's nothing after a return, and its not returning a specific value, you don't need a return at all.
+The . var stores the return of a function and will be returned even without a specific return call.
+***Good:***
+```
+/proc/do_thing()
+	do_thing
+	return result_of_doing_thing
+	
+/proc/do_thing()
+	do_thing
+	. = result_of_doing_thing
+	
+/proc/do_thing()
+	do_thing
+	do_other_thing
+```
+***Bad:***
+```
+/proc/do_thing()
+	do_thing
+	. = result_of_doing_thing
+	return
+	
+/proc/do_thing()
+	do_thing
+	do_other_thing
+	return
+```
+***
```

Comment:
Style does not match other parts of guidlines.

Reply:
in what manner?

Reply:
Sry that was problem with displaying indents in WebUI